src/Compilers/CSharp/Test/Syntax/LexicalAndXml/PreprocessorTests.cs

```diff
@@ -3425,18 +3425,18 @@ public void TestRegressPragmaWarningDisableWithWarningCode()
             var text = @"
 class A
 {
-    static void Main(bool b)
+    static void Main(int i)
     {
-#pragma warning disable 642
-        if (b);{}
+#pragma warning disable 1522
+        switch (i) { }
```

Comment:
Sorry, shoudl have explained.  This is a test that is validating that ```#pragma warning disable``` works at the syntax layer.  However, "if (b); {}" moved from being a warning produced in the syntax layer to one produced in the semantic layer.  As such the test failed.  As the test is about pragma warning disabling, and not about htat particular warning, i changed the test to another syntactic form that still reports the warning during parsing.